static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -72,6 +103,48 @@ function ReplayMetaData({replayRecord}: Props) {
   );
 }
 
+function StackedErrorCount({
+  errorCounts,
+}: {
+  errorCounts: Array<{count: number; project: Project | undefined}>;
```

Comment:
`undefined` is in here?

Reply:
return of `arr.find(predicate)` always returns `T | undefined`. it's gonna be extremely unlikely that it occurs, but at least the typing is there and we handle this case by rendering null